cob_bringup/robots/cob4-5.xml

```diff
@@ -131,6 +131,7 @@
 			<arg name="component_name" value="arm_left"/>
 			<arg name="can_device" value="can3"/>
 			<arg name="sim" value="$(arg sim)"/>
+			<arg if="$(arg sim)" name="cartesian_control" value="true"/>
```

Comment:
as long as we use the "old" `cob_obstacle_distance` we cannot use this on the real hardware because it leads to high load
however, I think `if="$(arg sim)"` is ok

but add it to to **both** arms!